schema.sql

```diff
@@ -1,5 +1,13 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    name varchar(100),
 );
+
+CREATE TABLE ANIMAL(id INTEGER,
```

Comment:
- [ ] [ **Optional**] I recommend you to make the id column not null and autoincremented, also set id as the primary key. In this way, you don't need to add the id to your data insertions.

> `id int NOT NULL IDENTITY(1,1)`
> `PRIMARY KEY (id)`